### PR TITLE
Multiple public keys code

### PR DESCRIPTION
**Pull request by evildvl**
Update README.md with the code example by @avimar (https://github.com/openpgpjs/openpgpjs/issues/271#issuecomment-274901731). Think that it's too hard to find and missed in documentation.

### CODE REVIEW

**Comment by sanjanarajan** [comment]
Could you please integrate this into the existing "Encrypt and decrypt String data with PGP keys" example? It's quite out of place as a standalone variable definition in between generation and revocation.

**Comment by evildvl** [comment]
Sure, branch updated.

**Comment by evildvl** [comment]
@avimar updated

File: README.md [hunk 1]
```diff
@@ -198,6 +198,42 @@ const encryptDecryptFunction = async() => {
...
+    const privKeyObj = (await openpgp.key.readArmored(privkey)).keys[0]
+    await privKeyObj.decrypt(passphrase)
+    
+    pubkeys = pubkeys.map(async (key) => {
+    	return (await openpgp.key.readArmored(key)).keys[0]
+    });
+    
+    const options = {
+        message: openpgp.message.fromText(message),       
+        publicKeys: pubkeys,           				  // for encryption
```
Comment (niponchi) [inline README.md]: From line 220, `pubkeys` is array of `Promise`, not array of `Key`.
In this case, line 226 should be `publicKeys: await Promise.all(pubkeys)`

Comment (evildvl) [reply README.md]: Helllo, I will check it and fix if here is the error

**Pull request merged**